ui/queue_wait.md

```diff
@@ -0,0 +1,74 @@
+### Running through the Queue and Busy Waiting
+
+General usecase for the UI is to display stuff from the database and queue
+request to the database.
+
+The state in the database is maintained by workers as well as various taks that
+are queued are handled by various workers.
+
+The UI calls methods on models or services and and those methods queue things
+that are then done by the workers and later the UI can see the updated state.
+
+Example: If we power off a VM we realy don't do the API call to RHEV or EC2 or
+VMware straight. But we queue duch action. In the UI we display a flash message
```

Comment:
Instead, we queue such an action and display flash message or notification in the UI.